tests/unit/test_images.py

```diff
@@ -156,3 +157,293 @@ def test_generate_json_ocr_url(code, image_id, flavor, environment, expected):
+                },
+            },
+            "uploaded_t": "1490702616",
+            "uploader": "user1",
+        },
+        "2": {
+            "sizes": {
+                "100": {
+                    "h": 100,
+                    "w": 56,
+                    "url": "https://images.openfoodfacts.org/images/products/326/385/950/6216/2.100.jpg",
+                },
+                "400": {
+                    "h": 400,
+                    "w": 225,
+                    "url": "https://images.openfoodfacts.org/images/products/326/385/950/6216/2.400.jpg",
+                },
+                "full": {
+                    "h": 3555,
+                    "w": 2000,
+                    "url": "https://images.openfoodfacts.org/images/products/326/385/950/6216/2.jpg",
+                },
+            },
+            "uploaded_t": "1490702690",
+            "uploader": "user2",
+        },
+        "3": {
+            "sizes": {
+                "100": {
+                    "h": 100,
+                    "w": 56,
+                    "url": "https://images.openfoodfacts.org/images/products/326/385/950/6216/3.100.jpg",
+                },
+                "400": {
+                    "h": 400,
+                    "w": 225,
+                    "url": "https://images.openfoodfacts.org/images/products/326/385/950/6216/3.400.jpg",
+                },
+                "full": {
+                    "h": 3555,
+                    "w": 2000,
+                    "url": "https://images.openfoodfacts.org/images/products/326/385/950/6216/3.jpg",
+                },
+            },
+            "uploaded_t": "1490702705",
+            "uploader": "user2",
+        },
+    },
+    "selected": {
+        "front": {
+            "fr": {
+                "imgid": "3",
+                "rev": "27",
+                "sizes": {
+                    "100": {
+                        "h": 100,
+                        "w": 75,
+                        "url": "https://images.openfoodfacts.org/images/products/326/385/950/6216/front_fr.27.100.jpg",
+                    },
+                    "200": {
+                        "h": 200,
+                        "w": 150,
+                        "url": "https://images.openfoodfacts.org/images/products/326/385/950/6216/front_fr.27.200.jpg",
+                    },
+                    "400": {
+                        "h": 400,
+                        "w": 300,
+                        "url": "https://images.openfoodfacts.org/images/products/326/385/950/6216/front_fr.27.400.jpg",
+                    },
+                    "full": {
+                        "h": 1200,
+                        "w": 901,
+                        "url": "https://images.openfoodfacts.org/images/products/326/385/950/6216/front_fr.27.full.jpg",
+                    },
+                },
+                "generation": {
+                    "white_magic": "0",
```

Comment:
I changed the type of the values of "white_magic" and "normalize" to boolean (real JSON boolean, not "true" string)